demisto_sdk/commands/unify/yml_unifier.py

```diff
@@ -465,10 +466,11 @@ def add_contributors_support(self, unified_yml: Dict, contributor_type: str,
 
         return unified_yml
 
-    def get_integration_doc_link(self, unified_yml: Dict) -> str:
+    def get_integration_doc_link(self, unified_yml: Dict, detailed_description='') -> str:
```

Comment:
tip: with your current implementation, you added non-trivial logic (and in particular a non-trivial argument) to the function. the best practice would be to keep the logic here as is, and when you **call** this function, check before if there is already a doc link in the current detailed description, and if not - call the function.